### PR TITLE
Naive implementation of the Votedelete feature.

### PR DESCRIPTION
**Pull request by kingslayer268**
For #41.

### CODE REVIEW

**Comment by DhanrajHira** [comment]
Is this formatted with black? We use the default config to format all our code in this repo.

**Comment by kingslayer268** [comment]
Nope, it hasn't been formatted using black. Couldn't find any code style guidelines on the repo but I did try to follow the code style from other files.

File: cogs/votedelete.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        self.votedelete_emoji_str = VOTEDELETE_EMOJI_STR
+        self.threshold = VOTEDELETE_THRESHOLD
+        self.ignored_channels = IGNORED_CHANNELS
+        self.bot = bot
+
+    async def cog_load(self):
+        await super().cog_load()
+        print("VoteDelete loaded.")
+
+    def checks(self, reaction):
```
Comment (Sooraj-beep) [inline cogs/votedelete.py]: maybe call this function `checkVoteDelete` or `checkDelete` just to make things a little clearer?

File: cogs/votedelete.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+class VoteDelete(commands.Cog):
+    def __init__(self, bot: discord.Client):
+        self.votedelete_emoji_str = VOTEDELETE_EMOJI_STR
+        self.threshold = VOTEDELETE_THRESHOLD
+        self.ignored_channels = IGNORED_CHANNELS
+        self.bot = bot
+
+    async def cog_load(self):
+        await super().cog_load()
+        print("VoteDelete loaded.")
```
Comment (Sooraj-beep) [inline cogs/votedelete.py:41]: When reaction count reaches threshold, delete message

```suggestion
        if reaction.count < self.threshold:
```

**Comment by Sooraj-beep** [comment]
@kingslayer268 what permissions does the bot need to be able to delete messages?

**Comment by kingslayer268** [comment]
@Sooraj-beep I'm pretty sure the manage messages permissions should be enough to delete the messages. Also, I've implemented your suggestions.

**Review by Sooraj-beep — APPROVED** [review]

**Pull request merged**